AutoCollection/HttpRequests.ts

```diff
@@ -8,6 +8,7 @@ import Util = require("../Library/Util");
 import RequestResponseHeaders = require("../Library/RequestResponseHeaders");
 import HttpRequestParser = require("./HttpRequestParser");
 import { CorrelationContextManager, CorrelationContext, PrivateCustomProperties } from "./CorrelationContextManager";
+import AutoColllectPerformance = require("./Performance");
```

Comment:
Minor: Looks like extra 'l' is in the name.

Reply:
Good catch!